parachain/pallets/vc-management/src/lib.rs

```diff
@@ -161,6 +161,12 @@ pub mod pallet {
 			detail: ErrorDetail,
 			req_ext_hash: H256,
 		},
+		VCHasBeenIssued {
```

Comment:
This definition is temporary, after we migrate the IDGraphs and sunset the old code we should have a single `VCIssued` event

Reply:
What's the consequence of directly changing it? I think we can make it work by firstly runtime-upgrade the chain, then the worker

Reply:
The only difference is that the current call and event expect `id_graph_hash` and this one `AccountId` (OmniAccount), we can directly update the current call and event but we also need to update the worker (old code) not to call `vc_issued`. The idea was that we can run both versions at the same time without modifying the current functionality

Reply:
I see, then maybe we can make it simple and call it `VcIssued2` or `VcIssuedNew` - I assume eventually this will replace the old call name and we are going to have only 1.

Also, we only need an OmniAccount identitifer and assertion, right? It seems we don't need the `Identity` and `reqExtHash`

Reply:
I guess `Identity` is probably not needed, I was just replicating what we currently have (in the tests we make assertions on the identity that requested the vc). `reqExtHash` is needed for the client to identify the event. 

I'll update the name of the event if you guys prefer it, the idea was to called as the original after the migration